src/main/application/schemas/slack_message.sd

```diff
@@ -0,0 +1,86 @@
+schema slack_message {
+
+    field language type string {
+        indexing: "en" | set_language
+    }
+
+    document slack_message {
+
+        field message_id type string {
+            indexing: summary | attribute | index
+        }
+
+        field thread_id type string {
+            indexing: summary | attribute | index
+            stemming: best
+        }
+
+        field text type string {
+            indexing: summary | attribute | index
```

Comment:
Remove attribute unless you want to group or sort on text.

Reply:
ok thanks, i removed it